### PR TITLE
[Docs] GitHub Actions

### PR DESCRIPTION
**Pull request by saurabh100ni**
**Notes for Reviewers**

this is the updated work from previous [PR](https://github.com/meshery/meshery/pull/9515) raised in [Meshery Repository ](https://github.com/meshery/meshery).

This PR fixes #
[#9440](https://github.com/meshery/meshery/issues/9440)

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: content/en/cloud/github-actions/_index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Meshery introduces the MeshMap Snapshot GitHub Actions, a tool that allows you to visualize changes in your infrastructure directly in your pull requests. With MeshMap Snapshot, you can:
+
+- See your deployment changes pull request-to-pull request.
+- Get snapshots of your infrastructure directly in your pull requests.
+
+### Getting ready for GitOps
+
+**Step 1: Get access to MeshMap**
+
+To initiate the GitOps workflow with Meshery, it is essential to [sign up](https://layer5.io/cloud-native-management/meshmap#:~:text=Signup%20for,MeshMap%20Beta) for MeshMap. MeshMap serves as a fundamental component, providing visual insights into infrastructure changes and facilitating the synchronization between the actual and desired states. Ensure that you have proper access to MeshMap to leverage its capabilities for effective GitOps implementation.
```
Comment (leecalcote) [inline content/en/cloud/github-actions/_index.md:21]: This link ultimately lands visitors on the wrong section of th page.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: content/en/cloud/github-actions/_index.md (hunk 1 above)
Comment (leecalcote) [reply content/en/cloud/github-actions/_index.md:21]: Make sense?

Comment (saurabh100ni) [reply content/en/cloud/github-actions/_index.md:21]: Yes, it does. Fixing it.

Comment (leecalcote) [reply content/en/cloud/github-actions/_index.md]: @saurabh100ni, tell me how you fixed it and ask for review (again) if you're sure that its ready.

Do this as a general behavior, too.

Comment (saurabh100ni) [reply content/en/cloud/github-actions/_index.md]: @leecalcote For some reason web-page for [meshmap](https://layer5.io/cloud-native-management/meshmap#:~:text=Signup%20for,MeshMap%20Beta) is navigating to `COLLABORATE` section by default.
So, I redirected it to the same form on [play.meshery.io](https://play.meshery.io/).

**Pull request merged**